Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -21,6 +23,16 @@
 static const int CACHE_VERSION = 13;  // Last changed in PR #3261
 static const int DATASTREAM_VERSION = QDataStream::Qt_5_5;
 
+u64 GetTitleID(const QString& path)
```

Comment:
Oh, seems like DolphinWX also does some volume creation to get the title ID when it would be simpler to just get it from the ISOFile... I'll have to change that sometime.